packages/systemd/systemd.spec

```diff
@@ -213,6 +213,9 @@ install -p -m 0644 %{S:3} %{buildroot}%{_cross_libdir}/systemd/journald.conf.d/j
 # with container networking by attempting to manage veth devices.
 rm -f %{buildroot}%{_cross_libdir}/systemd/network/*
 
+# Remove default target created by systemd
```

Comment:
```suggestion
# Remove default target created by systemd, since we override it elsewhere.
```

Reply:
Updated to mention that the default systemd unit is created in the release spec